lib/insulet/insuletSimulator.js

```diff
@@ -346,6 +346,7 @@ exports.make = function(config){
       // (although within their own type all the basals are always in order)
       // end result: we have to sort events again before we try to upload them
       var orderedEvents = _.sortBy(_.filter(events, function(event) {
+        delete event.index;
```

Comment:
Will add a comment, although this is going away very soon. Once we have UTC bootstrapping, I won't need the log index to get the right order of pump events.